otx/algorithms/classification/tasks/train.py

```diff
@@ -156,6 +156,10 @@ def _init_train_data_cfg(self, dataset: DatasetEntity):
                     otx_dataset=dataset.get_subset(Subset.VALIDATION),
                     labels=self._labels,
                 ),
+                unlabeled=ConfigDict(
+                    otx_dataset=dataset.get_subset(Subset.UNLABELED),
+                    labels=self._labels,
```

Comment:
It's just my curiosity. unlabled should need to get lables?

Reply:
Updated to set as `None`, thank you for comment :)